vanilla-framework/vanilla-framework-react
# 84: Rearchitect project to publish components to NPM

# Done
- Added missing exports to `src/lib/index.js`
- Renamed Row and Column to StripRow and StripColumn (as they are Strip children)
- Removed unnecessary scripts from `package.json` and changed build script
- Removed unnecessary build files (as this is a component library, not an app)
- Removed all `.scss` files and all references to them until a suitable method of encapsulating styles in components is proposed. This means that for now any projects using vanilla-framework-react MUST also have vanilla-framework.
- Added vanilla-framework hotlink to Storybook (so they're actually styled)
- Updated webpack configs
- Updated version to 0.1.0
- Added `.npmignore` so that only the build folder is published
- Updated `README.md`

# QA
- Pull code
- Run `yarn lint` and `yarn test` to ensure there are no linting or testing errors
- Run `./run serve --watch` and navigate to http://localhost:8102/
- Check that all the components still look and operate as expected (CodeBlock numbered requires a fix in Vanilla)
- Read the `README.md` and see if you can import the React components into your own project. Note: I've published to my own registry [here](https://www.npmjs.com/package/vanilla-framework-react-caleb) which is exactly the same as this PR (so when following the `README.md` replace all instances of vanilla-framework-react with vanilla-framework-react-caleb). I'll publish to the main registry once this is approved.
- I have an example test project [here](https://github.com/Caleb-Ellis/vfreact-test)

# Fixes
Fixes #84
